recipe/conda_build_config.yaml

```diff
@@ -12,6 +12,8 @@ build_type:
 channel_targets:
   - conda-forge main
   - conda-forge python_debug   # [not win]
+is_python_min:
+  - no
```

Comment:
Could we put this next to the other `python` keys [from `zip_keys`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/0c4535a2c15d4569f30a9395e2b93bdedf938ca4/recipe/conda_build_config.yaml#L186-L190 )? That would make it clearer this is related

https://github.com/conda-forge/python-feedstock/blob/99da098ce836c17458af50a8b622760a35d0ba3b/recipe/conda_build_config.yaml#L1-L6